ska_trend/fid_drop_mon/task_schedule.cfg

```diff
@@ -0,0 +1,56 @@
+# Configuration file for task_schedule.pl to run astromon jobs
+
+subject           Fid Drop Monitor
+timeout           2400               # Default tool timeout
+heartbeat_timeout 30000             # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     $ENV{SKA_DATA}/fid_drop_mon      # Data file directory
+log_dir      $ENV{SKA_DATA}/fid_drop_mon/logs  # Log file directory
+bin_dir      $ENV{SKA_SHARE}/fid_drop_mon     # Bin dir (optional, see task def'n)
```

Comment:
Generally the `SKA_*` env vars are deprecated and you should just use `$ENV{SKA}/data` etc. Those `SKA_DATA` vars were a great example of over-configuration (lesson learned, keep it simple!).

Reply:
Also nothing will be installed to `bin_dir` (`$SKA/share`) so probably just delete this line.